precompile/contracts/deployerallowlist/module.go

```diff
@@ -43,10 +44,10 @@ func (*configurator) MakeConfig() precompileconfig.Config {
 
 // Configure configures [state] with the given [cfg] precompileconfig.
 // This function is called by the EVM once per precompile contract activation.
-func (c *configurator) Configure(chainConfig precompileconfig.ChainConfig, cfg precompileconfig.Config, state contract.StateDB, blockContext contract.ConfigurationBlockContext) error {
+func (c *configurator) Configure(chainConfig precompileconfig.ChainConfig, cfg precompileconfig.Config, state contract.StateDB, blockNumber *big.Int) error {
```

Comment:
Could we keep both the block number and time here even if the time is unused? As we previously exposed this interface in `precompile-evm` and it seems it could be used in the future?

Reply:
Sure, one of the last commits removes the time argument, I wanted to see your opinion on this 😉

Reply:
Actually, similar question: what would be the cost to change the function signature in the future?

Reply:
It would require changes to all precompiles, both those that we have developed and any external code bases that may have developed precompiles for use in subnet-evm or precompile-evm.
So I think we should avoid changing it without good reason, but we can still change it to deliver improvements, features, and a better interface.

It seems natural the code that sets up the precompile would have access to both the timestamp and the number of the block where it's activated.

Reply:
Interesting, should we then not change the signature, such that the block context interface (now removed) can be later extended?

Reply:
No longer changing this